js/commonMenu.js

```diff
@@ -151,7 +151,7 @@ module.exports.preferencesMenuItem = {
 
 module.exports.bookmarksMenuItem = {
   label: 'Bookmarks manager...',
-  accelerator: 'CmdOrCtrl+Alt+b',
+  accelerator: process.platform === 'win32' ? 'Shift+Alt+B' : 'CmdOrCtrl+Alt+B',
```

Comment:
Actually Linux is always the same as Windows as well so this should be `const isDarwin = process.platform === 'darwin'` w/ an isDarwin ? X : Y.